### DISCUSSION TITLE
Meaning of Xt setting

### DISCUSSION THREAD

**Discussion by print118** [GuidodiPasquo/Actively-Stabilized-Model-Rocket-Simulator#14] [Q&A]
Hi, the Xt setting in the parameters window refer to _"the position of the TVC mount"_. What does that actually mean? Does the position mean the point where the TVC mount is connected to the rocket or does it refer to the rocket motors end or something else?

**Comment by GuidodiPasquo** [comment]
Hello!
It indeed is the point where the TVC mount is connected to the rocket.
